src/components/gltf-model-plus.js

```diff
@@ -168,7 +168,7 @@ const inflateEntities = function(node, templates, isRoot, modelToWorldScale) {
     for (const prop in entityComponents) {
       if (entityComponents.hasOwnProperty(prop) && AFRAME.GLTFModelPlus.components.hasOwnProperty(prop)) {
         const { componentName, inflator } = AFRAME.GLTFModelPlus.components[prop];
-        inflator(el, componentName, entityComponents[prop]);
+        inflator(el, componentName, entityComponents[prop], entityComponents);
```

Comment:
Still would prefer we not pass an element of a collection AND the whole collection into a function like this

Reply:
Our inflator inflates multiple components that are dependent on one another `networked` and `media-loader`. In glTF I also represent these as two components `image`/`video` and `networked` I use the id stored in the networked component when adding the `networked` component to the entity. I somewhat agree that it isn't ideal that these cant be separate concerns. However, I do not know of a better alternative.

Reply:
I just meant since we are passing in the collection we don't need to pass in the single item, just pass in the collection:
```
inflator(el, componentName, entityComponents); 
```